### PR TITLE
[SIGNUP] user can signup

### PR DESCRIPTION
**Pull request by purnima-nyc**
## Description
- User can signup 
- CircleCi runs mix format

https://trello.com/c/z1Tqm9Tt
https://trello.com/c/YF2Qrj7v

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,5 +52,7 @@ jobs:
 
       - run: mix test 
 
+      - run: mix format --check-formatted
```
Comment (ssunday) [inline .circleci/config.yml]: :clap:

File: lib/ebaze_web/controllers/user_controller.ex [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
     render(conn, "index.html", users: users)
   end
 
   def new(conn, _params) do
-    changeset = Accounts.change_user(%User{})
-    render(conn, "new.html", changeset: changeset)
+    if Guardian.Plug.current_resource(conn) do
+      conn
+      |> put_flash(:info, "You are already signed in!")
+      |> redirect(to: Routes.page_path(conn, :index))
```
Comment (ssunday) [inline lib/ebaze_web/controllers/user_controller.ex]: nice!

File: lib/ebaze_web/templates/user/form.html.eex [hunk 3]
```diff
@@ -10,7 +10,7 @@
   <%= error_tag f, :username %>
 
   <%= label f, :password %>
-  <%= text_input f, :password %>
+  <%= password_input f, :password %>
```
Comment (ssunday) [inline lib/ebaze_web/templates/user/form.html.eex]: card idea: for new user sign-up, have a confirmation field

Comment (purnima-nyc) [reply lib/ebaze_web/templates/user/form.html.eex]: https://trello.com/c/NtnlLdjO

**Comment by purnima-nyc** [comment]
Signup form
Screen Shot 2019-09-10 at 9 41 24 AM

User created successfully
Screen Shot 2019-09-10 at 9 41 36 AM

Signup redirects to index if user is already logged in
Screen Shot 2019-09-10 at 9 41 55 AM

**Pull request merged**